tools/gulp/tasks/docs.ts

```diff
@@ -69,10 +69,25 @@ task('docs', [
 
 /** Generates html files from the markdown overviews and guides. */
 task('markdown-docs', () => {
+  // Extend the renderer for custom heading anchor rendering
+  markdown.marked.Renderer.prototype.heading = (text: string, level: number): string => {
```

Comment:
The docs for marked give a different API for customizing a renderer, don't think you're supposed to monkey-patch the renderer prototype:
https://github.com/chjj/marked#overriding-renderer-methods